### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by LibE4**
fun js practice

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	divElement.innerHTML = "";
+} //end of clearTextarea function
+
+
+
+//get element for html display
+var divElement = document.getElementById("chartree");
+var buttonGrowTree = document.getElementById("growtree");
+var buttonClear = document.getElementById("clearinputbox");
+// var textElement = document.getElementsByClassName("input");
```
Comment (JoeShep) [inline main.js]: Remember to remove commented-out code before submitting a PR

File: main.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
+function growtree (clickEvent) {
+	// set object
+	var tree = new Object ();
```
Comment (JoeShep) [inline main.js]: This object needs to be created elsewhere and passed into the growTree function, per the requirements

**Comment by JoeShep** [comment]
Bin, this demonstrates an excellent grasp of the early JS material. Nice work. I've not seen this exact approach before, but it works well. Address the couple of small issues noted in the code and I'll be ready to give this a :+1:

**Comment by LibE4** [comment]
Joe, all current issues fixed. Waiting for new directions.

**Comment by JoeShep** [comment]
You're good to go 👍

**Pull request merged**